File: SeleniumWrapper/src/fr/lip6/segmentations/Document.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

package fr.lip6.segmentations;

import java.util.ArrayList;



/**
 *
 * @author sanojaa
 */
public class Document {
    private long w = 0;
    private long h = 0;
    
    public void parse(Object obj) {
        ArrayList harr = (ArrayList) obj;
        this.w = Long.parseLong(harr.get(0).toString());
        this.h = Long.parseLong(harr.get(1).toString());
    }
    public String toRecord() {
        return(this.w + "," + this.h);
    }
}

File: blockfusion/document.rb
class Document
	attr_accessor :url
	def initialize
		@w = 0
		@h = 0;
		@url = ""
	end

	def parse(harr)
		@w = harr['w']
		@h = harr['h']
	end
	
	def to_record
		"#{@w},#{@h}"
	end
end
